HomeTask3 v2.0/HomeTask3 v2.0/Program.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace HomeTask3_v2._0
+{
+    class Program
+    {
+        static void Main()
+        {
+         Console.WriteLine($"Значение x для варианта 16 равно {Method16(2, 3) + Method16(3, 7) + Method16(5, 5)}");
+        }
+        static double Method16(double x, double y)
+        {
+         return Math.Round(Math.Sqrt(x + Math.Log(1 + Math.Sqrt(y))),3);
```

Comment:
Округлять не нужно.
Вычисление значения должно быть точным.

> Результат **выведите на консоль** с точностью до трех знаков после запятой.

А не "**вычислите значение** с точностью до трех знаков после запятой"
Чувствуете разницу в формулировках?
```suggestion
         return Math.Sqrt(x + Math.Log(1 + Math.Sqrt(y)));
```